ui/src/alerting/utils/statusEvents.ts

```diff
@@ -0,0 +1,75 @@
+// Utils
+import {runQuery} from 'src/shared/apis/query'
+import {fromFlux} from '@influxdata/giraffe'
+
+// Constants
+import {MONITORING_BUCKET} from 'src/alerting/constants'
+
+// Types
+import {CancelBox, StatusRow, File} from 'src/types'
+import {RunQueryResult} from 'src/shared/apis/query'
+import {Row} from 'src/eventViewer/types'
+
+export const runStatusesQuery = (
+  orgID: string,
+  checkID: string,
+  extern: File
+): CancelBox<StatusRow[][]> => {
+  const query = `
+from(bucket: "${MONITORING_BUCKET}")
+  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+  |> filter(fn: (r) => r._measurement == "statuses" and r._field == "_message")
+  |> keep(columns: ["_time", "_value", "_check_id", "_check_name", "_level"])
+  |> rename(columns: {"_time": "time",
```

Comment:
*nit* If we put the rename at the very end of things, I think it cleans things up a tiny bit.


```
from(bucket: "${MONITORING_BUCKET}")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r._measurement == "statuses" and r._field == "_message")
  |> keep(columns: ["_time", "_value", "_check_id", "_check_name", "_level"])
  |> window(every: 1s)
  |> group(columns: ["_start", "_stop"])
  |> filter(fn: (r) => r.check_id == "${checkID}")
  |> rename(columns: {"_time": "time",
                      "_value": "message",
                      "_check_id": "checkID",
                      "_check_name": "checkName",
                      "_level": "level"})
```